src/networks.js

```diff
@@ -22,7 +27,28 @@ module.exports = {
     },
     pubKeyHash: 0x6f,
     scriptHash: 0xc4,
-    wif: 0xef
+    wif: 0xef,
+    forkHeight: 1,
+    equihash: {
+      n: 48,
```

Comment:
Testnet's Equihash parameters are the same as mainnet: (200, 9)

Reply:
Fixed